webportal/package.json

```diff
@@ -67,18 +68,14 @@
     "winston": "2.4.0"
   },
   "scripts": {
-    "build:dev": "npm run clean && npm run preprocess && npm run webpack -- --watch",
-    "build": "npm run clean && npm run preprocess && npm run webpack",
-    "clean:dist": "npm run rimraf -- dist",
-    "clean": "npm cache clean && npm run clean:dist",
     "lint": "eslint \"src/**/*.js\"",
-    "node": "node",
-    "preprocess": "node config/preprocess.js",
-    "rimraf": "rimraf",
-    "server": "node server/index.js",
-    "start": "npm run build && npm run server",
     "test": "npm run lint",
-    "webpack": "webpack --config webpack.config.js --progress --profile",
+    "clean": "rimraf dist",
+    "prebuild": "npm run clean && node config/preprocess.js",
```

Comment:
When are `prebuild` and `prestart` called?

Reply:
@hwuu They runs just before `npm run build` & `npm start` (shorthand of `npm run start`)

<https://docs.npmjs.com/misc/scripts>